### PR TITLE
Added .gitignore and .editorconfig files

### PR DESCRIPTION
**Pull request by pritamnegi**
#13

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+.spyproject
+
+# Rope project settings
+.ropeproject
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
```
Comment (satwikkansal) [inline .gitignore]: Please add a newline at the end. It's a common standard to end files with a newline. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+# top-most EditorConfig file
+root = true
+
+# defaults
+[*]
+insert_final_newline = true     
+
+# 4 space indentation
+[*.{ini,py,py.tpl,rst}]
```
Comment (satwikkansal) [inline .editorconfig]: We don't have any `ini` or `.tpl` or `.rst`  files currently in the project.

Comment (satwikkansal) [reply .editorconfig]: Please cross check what all files do we have currently in the repo, and adapt accordingly

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+indent_style = space
+indent_size = 4
+
+# 4-width tabbed indentation
+[*.{sh,bat.tpl,Makefile.tpl}]
```
Comment (satwikkansal) [inline .editorconfig]: Again, there are no such files present. We can add such entries later once we have these files in the codebase. I hope you get the idea :)

**Review by satwikkansal — CHANGES_REQUESTED** [review]
Nice works, few comments. 

Ping once addressed, and we're good to go!

**Comment by satwikkansal** [comment]
It seems that your commit message doesn't contain enough information. Don't worry, most people don't know about good commit messages initially, including very experienced developers :) Standards vary from project to project.

The most important part is the first line of your commit message - we can use it to read through a large number of commits and thus it's important that it's short but most importantly concise. I'd heartily recommend reading http://coala.io/commit !

To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.

Cheers!

**Comment by pritamnegi** [comment]
Please now review my PR. I have made the changes requested by mentors.

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: http://EditorConfig.org
```
Comment (satwikkansal) [inline .editorconfig:1]: We don't need this line :p

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+# defaults
+[*]
+insert_final_newline = true     
+
+# 4 space indentation
+[*.{py}]
+indent_style = space
+indent_size = 4
+
```
Comment (satwikkansal) [inline .editorconfig:14]: This is one extra line than blank line :)

**Review by satwikkansal — CHANGES_REQUESTED** [review]
A couple of changes in .editorconfig. Otherwise we're good to go :+1:

**Comment by satwikkansal** [comment]
Also, did you see https://github.com/wtfpython-web/wtfpython-web/pull/15#issuecomment-399772374 ?

**Comment by pritamnegi** [comment]
In what way I should change my commit message??

**Comment by satwikkansal** [comment]
Commit 1
```
.editorconfig: Add .editorconfig

Adds the .editorconfig file for the project.

Related to https://github.com/wtfpython-web/wtfpython-web/issues/13
```

Commit 2

```
.gitignore: Add .gitignore

Adds the .gitignore file for the project referenced from https://github.com/github/gitignore/blob/master/Python.gitignore

Closes https://github.com/wtfpython-web/wtfpython-web/issues/13
```

**Comment by realslimshanky** [comment]
@pritamnegi I suggest you take a look at Github's [gitignore](https://github.com/github/gitignore) as well to check if we need something more for our project. Also, make sure your PR description is informative enough.

**Comment by pritamnegi** [comment]
Ok as you say @realslimshanky

**Pull request closed without merging**